microsoft/testsuites/performance/common.py

```diff
@@ -494,6 +497,40 @@ def perf_iperf(
         notifier.notify(iperf3_message)
 
 
+def perf_sockperf(
+    test_result: TestResult, mode: str, test_case_name: str, set_busy_poll: bool = False
+) -> None:
+    environment = test_result.environment
+    assert environment, "fail to get environment from testresult"
+
+    client = cast(RemoteNode, environment.nodes[0])
+    server = cast(RemoteNode, environment.nodes[1])
+
+    if set_busy_poll:
+        for node in [client, server]:
+            node.tools[Sysctl].enable_busy_polling("50")
+
+    run_in_parallel([lambda: client.tools[Sockperf], lambda: server.tools[Sockperf]])
+
+    # disable any firewalls running which might mess with the test
+    for firewall in [client.tools[Firewall], server.tools[Firewall]]:
+        firewall.stop()
```

Comment:
can we move this to sockperf  tool, if it is needed when use this tool

Reply:
Works for me :) fixed